### PR TITLE
Connect grafana postgres db

### PR DESCRIPTION
**Pull request by oddscenes**
- Adds the postgres db to grafana as a data source. The data source is added via provisioning setup in grafana, which connects it automatically on startup
- Add config options for grafana via custom config file `./grafana/grafana.config` - we can modify custom options for grafana here
- Scaffolding setup for dashboards (`'./grafana/dashboards` & `'./grafana/provisioning/dashboard.yml`). No actual dashboards exist yet though

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,9 +21,9 @@ services:
       - 15672:15672
       - 5672:5672
     volumes:
-      - /tmp/rabbitmq/etc/:/tmp/etc/rabbitmq/
-      - /tmp/rabbitmq/data/:/var/lib/rabbitmq/
-      - /tmp/rabbitmq/logs/:/var/log/rabbitmq/
+      - ./tmp/rabbitmq/etc/:/tmp/etc/rabbitmq/
```
Comment (UltimateBeaver) [inline docker-compose.yml:24]: What're these changes for?

File: docker-compose.yml [hunk 2]
```diff
@@ -35,6 +36,27 @@ services:
...
+      - postgresdb
+    environment:
+      # GF_DATASOURCES_NAME: Postgres
+      # GF_DATASOURCES_TYPE: postgres
+      # GF_DATASOURCES_URL: localhost:5432
+      # GF_DATASOURCES_DATABASE: lake
+      # GF_DATASOURCES_USER: postgres
+      # GF_DATASOURCES_SECUREJSONDATA_PASSWORD: "postgres"
+      GF_DATABASE_TYPE: postgres
+      GF_DATABASE_HOST: postgresdb:5432
```
Comment (UltimateBeaver) [inline docker-compose.yml:53]: What was the fix to our issue yesterday?

**Review by UltimateBeaver — COMMENTED** [review]
Looking good in general, just some minor questions, please see my individual comments.

File: docker-compose.yml (hunk 2 above)
Comment (oddscenes) [reply docker-compose.yml:53]: The mapping here in grafana/volumes/provisioning is setup to a specific folder structure. That connects to `./grafana/provisioning/datasources/datasource.yml`

File: docker-compose.yml (hunk 1 above)
Comment (oddscenes) [reply docker-compose.yml]: This shouldn't have a `'./` - reverted it back to without (same as before)

**Pull request merged**